server.js

```diff
@@ -1,10 +1,11 @@
-const express = require("express");
+const express = require('express');
+const fallback = require('express-history-api-fallback');
 
 const app = express();
 const PORT = 3000;
```

Comment:
Бывает полезно (особенно при разворачивании приложение в контейнере на разных машинах) указать порт через переменные окружения
```suggestion
const PORT = process.env.PORT || 3000;
```
На данном этапе это не особо важно, но на будущее полезно знать зачем это делается